pkg/xds/envoy/clusters/common_lb_configurer.go

```diff
@@ -0,0 +1,25 @@
+package clusters
+
+import (
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+)
+
+func CommonLb() ClusterBuilderOpt {
```

Comment:
how about `LocalityWeightedLb()` because this is what we are setting essentially. When reading outbound proxy generator I'd prefer to read `LocalityWeightedLb()` instead of cryptic `CommonLb()`